services/streammanager/googlepubsub/googlepubsubmanager.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 type Config struct {
-	Credentials     string              `json:"credentials"`
-	ProjectId       string              `json:"projectId"`
-	EventToTopicMap []map[string]string `json:"eventToTopicMap"`
+	Credentials         string              `json:"credentials"`
+	ProjectId           string              `json:"projectId"`
+	EventToTopicMap     []map[string]string `json:"eventToTopicMap"`
+	EventToAttributeMap []map[string]string `json:"eventToAttributeMap"`
```

Comment:
@arajguha  remove this if it's not being used anywhere ?

Reply:
yes. I will remove that.